README.md

```diff
@@ -88,7 +88,7 @@ package3/package.json
 Output:
 
 ```pt
-Found 1 dependency with mismatching versions across the workspace. Fix with `--fix`.
+Found 1 dependency with mismatching versions across the workspace. Upgrade all to latest present version with `--fix`.
```

Comment:
We describe this as "highest" in other places so let's say "highest" instead of "latest". 

```
highest version present
```

Reply:
yarn uses "latest" in its documentation so this should be consistent with how upgrades are handled.

Reply:
Okay then can we use "latest version present" here and update the other references too (separate PR okay)?